vms/platformvm/vm_test.go

```diff
@@ -1167,79 +1168,28 @@ func TestBootstrapPartiallyAccepted(t *testing.T) {
 		MinStakeDuration:       defaultMinStakingDuration,
 		MaxStakeDuration:       defaultMaxStakingDuration,
 		RewardConfig:           defaultRewardConfig,
-		UpgradeConfig:          upgradetest.GetConfigWithUpgradeTime(upgradetest.Durango, latestForkTime),
```

Comment:
Q: Is there a reason we kept this as Durango and not latest before? I'm wondering if we should update the other tests in the file to also be the latest fork, or fine to keep them as is.